templates/systemd-system-omero-server-service.j2

```diff
@@ -10,6 +10,10 @@ After=postgresql-9.6.service
 Requires=network.service
 {% endif %}
 After=network.service
+{% for value in ((omero_server_systemd_after | default({})) | sort) %}After={{ value }}
```

Comment:
These are top-level vars, can you put them in `defaults.yml` instead? Also the default should be `[]` not `{}`, and since it's a list you don't need the sort, leave it in whatever order the user provided.

Reply:
How's https://github.com/openmicroscopy/ansible-role-omero-server/pull/30/commits/01b7629aef129fb1dd8c54b6e72119e493d99b49 @manics ?